components/brave_wallet/browser/brave_wallet_prefs.cc

```diff
@@ -88,6 +88,8 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
   p3a_utils::RegisterFeatureUsagePrefs(
       registry, kBraveWalletP3AFirstUnlockTime, kBraveWalletP3ALastUnlockTime,
       kBraveWalletP3AUsedSecondDay, nullptr, nullptr);
+  registry->RegisterBooleanPref(kBraveWalletP3ANewUserBalanceReported, false);
+  registry->RegisterIntegerPref(kBraveWalletP3AOnboardingLastStep, -1);
```

Comment:
Let's not use -1 as default. If pref was never set or was cleared HasPrefPath(kBraveWalletP3AOnboardingLastStep) should return false.